### PR TITLE
updated readme & add db.dump

### PR DESCRIPTION
**Pull request by GeorgStrassberger**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,14 +5,12 @@
...
 .vscode
 
-
 ### Django ###
 *.log
 *.pot
 *.pyc
 __pycache__/
 local_settings.py
-db.sqlite3
```
Comment (mihaiandreineacsu) [inline .gitignore:15]: Please do not commit database to version control. Ignore this database file in `.gitignoire`.

File: README.md [hunk 2]
```diff
@@ -10,60 +30,82 @@ The project demonstrates:
...
 - Docker (latest version)
 
-### Technologies
+---
+
+## Technologies
 
 - Python 3.9
 - Django 4.0.2
 - Venv
```
Comment (mihaiandreineacsu) [inline README.md:48]: Better will be `Python virtual environment`.

File: README.md [hunk 3]
```diff
@@ -10,60 +30,82 @@ The project demonstrates:
...
 ```bash
-  docker run -d -p 8025:5000 --name babyshop-app --restart always babyshop
+  docker volume create babyshop-data
 ```
 
-- `-d | detached` Runs the container in the background
+##### 4. Create a Docker Container from the Image
+
+```bash
+  docker run -d -p 8025:8000 --name babyshop-app -v babyshop-data:/babyshop --restart always babyshop
```
Comment (mihaiandreineacsu) [inline README.md:83]: Example of formatting long commands:

For bash
```bash
  docker run -d \
    -p 8025:8000 \
    --name babyshop-app \
    -v babyshop-data:/babyshop \
    --restart always \
    babyshop
```

For Powershell

```powershell
  docker run -d `
    -p 8025:8000 `
    --name babyshop-app `
    -v babyshop-data:/babyshop `
    --restart always `
    babyshop
```

**Review by mihaiandreineacsu — COMMENTED** [review]
Please apply the changes that follow.

**Comment by GeorgStrassberger** [comment]
Hey danke, setz ich gleich um

**Pull request merged**